### DISCUSSION TITLE
Error cloning http repository

### DISCUSSION THREAD

**Discussion by root3200** [semaphoreui/semaphore#2061] [Q&A] [answered]
Hello.

In the new version 2.9.109 the http protocol is implemented, I'm using it to clone my private repositories but only works when the repository is public, in private I get the following error .

![Captura de pantalla 2024-05-27 125616](https://github.com/semaphoreui/semaphore/assets/58263431/f035f64a-b2e3-493c-af4f-a66cfd5fff96)

Researching I found that there is a how fix, but I think it does not apply to the http protocol.

https://docs.semui.co/administration-guide/troubleshooting#fatal-bad-numeric-config-value-0-for-git_terminal_prompt-invalid-unit

**Comment by jengstrom440** [comment]
I can add my experiences with it, as I was not able to get https to work unless it was a public repo and not private. I am curious to see if you can create a test dummy project that is set to public and test it there just to see if it works? Switching to ssh worked for me as I couldn't get https to work at all unless it was a public repo and not private.

**Comment by fiftin** [comment, marked as answer]
Hi All, it is looks like a bug. I created an issue: https://github.com/semaphoreui/semaphore/issues/2071
